<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="twitter:card" content="summary_large_image" /><!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Small Molecules Resources | Pushkar G. Ghanekar</title>
<meta name="generator" content="Jekyll v4.1.1" />
<meta property="og:title" content="Small Molecules Resources" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Compendium of recent articles, resources, and blogs in the area of chemical science and drug discovery" />
<meta property="og:description" content="Compendium of recent articles, resources, and blogs in the area of chemical science and drug discovery" />
<link rel="canonical" href="http://pgg1610.github.io/blog_fastpages/chemical-science/machine-learning/resources/2022/12/28/Computational_drug_discovery.html" />
<meta property="og:url" content="http://pgg1610.github.io/blog_fastpages/chemical-science/machine-learning/resources/2022/12/28/Computational_drug_discovery.html" />
<meta property="og:site_name" content="Pushkar G. Ghanekar" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2022-12-28T00:00:00-06:00" />
<script type="application/ld+json">
{"url":"http://pgg1610.github.io/blog_fastpages/chemical-science/machine-learning/resources/2022/12/28/Computational_drug_discovery.html","@type":"BlogPosting","headline":"Small Molecules Resources","dateModified":"2022-12-28T00:00:00-06:00","datePublished":"2022-12-28T00:00:00-06:00","mainEntityOfPage":{"@type":"WebPage","@id":"http://pgg1610.github.io/blog_fastpages/chemical-science/machine-learning/resources/2022/12/28/Computational_drug_discovery.html"},"description":"Compendium of recent articles, resources, and blogs in the area of chemical science and drug discovery","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/blog_fastpages/assets/css/style.css"><link type="application/atom+xml" rel="alternate" href="http://pgg1610.github.io/blog_fastpages/feed.xml" title="Pushkar G. Ghanekar" /><!-- the google_analytics_id gets auto inserted from the config file -->



<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-167983168-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-167983168-2');
</script>


<link rel="shortcut icon" type="image/x-icon" href="/blog_fastpages/images/favicon_glass.ico"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Primer/15.2.0/primer.css" integrity="sha512-xTz2ys4coGAOz8vuV1NcQBkgVmKhsSEtjbqyMJbBHRplFuvKIUo6xhLHpAyPt9mfR6twHJgn9OgVLuqOvjeBhg==" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.12.0/katex.min.css" integrity="sha512-h7nl+xz8wgDlNM4NqKEM4F1NkIRS17M9+uJwIGwuo8vGqIl4BhuCKdxjWEINm+xyrUjNCnK5dCrhM0sj+wTIXw==" crossorigin="anonymous" />
    <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.12.0/katex.min.js" integrity="sha512-/CMIhXiDA3m2c9kzRyd97MTb3MC6OVnx4TElQ7fkkoRghwDf6gi41gaT1PwF270W6+J60uTmwgeRpNpJdRV6sg==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.12.0/contrib/auto-render.min.js" integrity="sha512-Do7uJAaHZm5OLrIv/yN4w0iG1dbu01kzdMNnFfu/mAqgUk6Nniv2JYHcwH+cNwjqgLcqcuBBk+JRvprLVI8azg==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A==" crossorigin="anonymous"></script>
    <script>
    document.addEventListener("DOMContentLoaded", function() {
        renderMathInElement( document.body, {
        delimiters: [
            {left: "$$", right: "$$", display: true},
            {left: "[%", right: "%]", display: true},
            {left: "$", right: "$", display: false}
        ]}
        );
    });
    </script>


<script>
function wrap_img(fn) {
    if (document.attachEvent ? document.readyState === "complete" : document.readyState !== "loading") {
        var elements = document.querySelectorAll(".post img");
        Array.prototype.forEach.call(elements, function(el, i) {
            if (el.getAttribute("title") && (el.className != "emoji")) {
                const caption = document.createElement('figcaption');
                var node = document.createTextNode(el.getAttribute("title"));
                caption.appendChild(node);
                const wrapper = document.createElement('figure');
                wrapper.className = 'image';
                el.parentNode.insertBefore(wrapper, el);
                el.parentNode.removeChild(el);
                wrapper.appendChild(el);
                wrapper.appendChild(caption);
            }
        });
    } else { document.addEventListener('DOMContentLoaded', fn); }
}
window.onload = wrap_img;
</script>

<script>
    document.addEventListener("DOMContentLoaded", function(){
    // add link icon to anchor tags
    var elem = document.querySelectorAll(".anchor-link")
    elem.forEach(e => (e.innerHTML = '<i class="fas fa-link fa-xs"></i>'));
    });
</script>
</head>
<body><header class="site-header">

  <div class="wrapper"><a class="site-title" rel="author" href="/blog_fastpages/">Pushkar G. Ghanekar</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/blog_fastpages/about/">About Me</a><a class="page-link" href="/blog_fastpages/search/">Search</a><a class="page-link" href="/blog_fastpages/categories/">Tags</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">Small Molecules Resources</h1><p class="page-description">Compendium of recent articles, resources, and blogs in the area of chemical science and drug discovery</p><p class="post-meta post-meta-title"><time class="dt-published" datetime="2022-12-28T00:00:00-06:00" itemprop="datePublished">
        Dec 28, 2022
      </time>
       • <span class="read-time" title="Estimated read time">
    
    
      42 min read
    
</span></p>

    
      <p class="category-tags"><i class="fas fa-tags category-tags-icon"></i></i> 
      
        <a class="category-tags-link" href="/blog_fastpages/categories/#chemical-science">chemical-science</a>
        &nbsp;
      
        <a class="category-tags-link" href="/blog_fastpages/categories/#machine-learning">machine-learning</a>
        &nbsp;
      
        <a class="category-tags-link" href="/blog_fastpages/categories/#resources">resources</a>
        
      
      </p>
    

    </header>

  <div class="post-content e-content" itemprop="articleBody">
    <ul class="section-nav">
<li class="toc-entry toc-h2"><a href="#noteworthy-blogs-to-follow">Noteworthy blogs to follow:</a></li>
<li class="toc-entry toc-h2"><a href="#online-resources">Online resources</a></li>
<li class="toc-entry toc-h2"><a href="#books">Books</a></li>
<li class="toc-entry toc-h2"><a href="#best-practices">Best practices</a></li>
<li class="toc-entry toc-h2"><a href="#reviews">Reviews</a></li>
<li class="toc-entry toc-h2"><a href="#industry-focused-drug-discovery-reviews">Industry-focused drug discovery reviews</a></li>
<li class="toc-entry toc-h2"><a href="#special-journal-issues">Special Journal Issues</a></li>
<li class="toc-entry toc-h2"><a href="#meeting-notes">Meeting notes</a></li>
<li class="toc-entry toc-h2"><a href="#chemical-modalities">Chemical modalities</a></li>
<li class="toc-entry toc-h2"><a href="#meta-themes-on-optimizing-small-molecules">Meta themes on optimizing small molecules</a></li>
<li class="toc-entry toc-h2"><a href="#synthesis-chemistry">Synthesis Chemistry</a></li>
<li class="toc-entry toc-h2"><a href="#binding-free-energetic-calculations">Binding free energetic calculations</a></li>
<li class="toc-entry toc-h2"><a href="#cheminformatics-focus">Cheminformatics-focus</a>
<ul>
<li class="toc-entry toc-h3"><a href="#representation">Representation</a></li>
<li class="toc-entry toc-h3"><a href="#predictive-modeling">Predictive modeling</a></li>
<li class="toc-entry toc-h3"><a href="#qsar-benchmarks">QSAR benchmarks</a></li>
<li class="toc-entry toc-h3"><a href="#enumeration-of-chemical-space">Enumeration of chemical space</a></li>
<li class="toc-entry toc-h3"><a href="#explainableinterpretable-machine-learning">Explainable/Interpretable Machine Learning</a></li>
<li class="toc-entry toc-h3"><a href="#uncertainty-quantification">Uncertainty quantification</a></li>
<li class="toc-entry toc-h3"><a href="#active-learning">Active Learning</a></li>
<li class="toc-entry toc-h3"><a href="#transfer-learning">Transfer Learning</a></li>
<li class="toc-entry toc-h3"><a href="#meta-learning">Meta Learning</a></li>
<li class="toc-entry toc-h3"><a href="#federated-learning">Federated Learning</a></li>
<li class="toc-entry toc-h3"><a href="#generative-models">Generative models</a></li>
<li class="toc-entry toc-h3"><a href="#computer-aided-synthesis-planning-casp">Computer Aided Synthesis Planning (CASP)</a></li>
<li class="toc-entry toc-h3"><a href="#data-driven-chemistry-modeling-and-reaction-optimization">Data-driven chemistry modeling and reaction optimization</a></li>
<li class="toc-entry toc-h3"><a href="#automated-chemistry-workflows">Automated chemistry workflows</a></li>
<li class="toc-entry toc-h3"><a href="#dna-encoded-libraries">DNA-encoded Libraries</a></li>
</ul>
</li>
<li class="toc-entry toc-h2"><a href="#code--packages">Code / Packages:</a></li>
<li class="toc-entry toc-h2"><a href="#datasets--chemical-libraries">Datasets &amp; Chemical libraries</a></li>
<li class="toc-entry toc-h2"><a href="#helpful-utilities">Helpful utilities:</a></li>
</ul><p>Last update: 28th December 2022</p>

<h2 id="noteworthy-blogs-to-follow">
<a class="anchor" href="#noteworthy-blogs-to-follow" aria-hidden="true"><span class="octicon octicon-link"></span></a>Noteworthy blogs to follow:</h2>

<ol>
  <li>
<a href="https://practicalcheminformatics.blogspot.com/2021/01/ai-in-drug-discovery-2020-highly.html">Patrick Walters Blog on Cheminformatics</a>
    <ul>
      <li><a href="https://github.com/PatWalters/resources/blob/main/cheminformatics_resources.md">Pat Walter’s Cheminformatics Resources list</a></li>
    </ul>
  </li>
  <li>
    <p><a href="https://iwatobipen.wordpress.com/">Is Life Worth Living</a></p>
  </li>
  <li>
    <p><a href="https://whitead.github.io/dmol-book/intro.html">Andrew White’s ML for Molecules and Materials Online Book</a></p>
  </li>
  <li>
    <p><a href="http://www.cheminformania.com">Cheminformia</a></p>
  </li>
  <li>
    <p><a href="https://depth-first.com">Depth-First</a></p>
  </li>
  <li>
    <p><a href="http://www.drugdiscovery.net/2019/07/25/new-post1/">DrugDiscovery.NET - Andreas Bender</a></p>
  </li>
  <li>
    <p><a href="https://drughunter.com">DrugHunter - Dennis Hu</a></p>
  </li>
  <li>
    <p><a href="http://practicalfragments.blogspot.com/">Practical Fragments</a></p>
  </li>
  <li><a href="https://www.science.org/blogs/pipeline">Derek Lowe’s In the Pipeline</a></li>
</ol>

<h2 id="online-resources">
<a class="anchor" href="#online-resources" aria-hidden="true"><span class="octicon octicon-link"></span></a>Online resources</h2>

<ul>
  <li>
    <p><a href="https://github.com/volkamerlab/TeachOpenCADD">Andrea Volkmer, TeachOpenCADD: a teaching platform for computer-aided drug design (CADD)</a> - Highly recommended.</p>
  </li>
  <li>
    <p><a href="https://github.com/PatWalters/practical_cheminformatics_tutorials">Patrick Walter’s Cheminformatics Tutorials</a></p>
  </li>
  <li>
    <p><a href="https://www.macinchem.org/blog/files/fe66130c1da3375e46d0512e483eb901-2791.php?utm_source=pocket_mylist">Pat Walters’ RSC CICAG Open Source Tools for Chemistry</a>.<a href="https://www.youtube.com/watch?v=2ZjerAGS_IQ">Video</a>. <a href="https://github.com/PatWalters/chem_tutorial">Github</a></p>
  </li>
  <li>
    <p><a href="https://github.com/iwatobipen/py4chemoinformatics">Pen’s Python cookbook for Cheminformatics</a></p>
  </li>
  <li>
    <p><a href="https://bit.ly/2SxItoc">Chem LibreText collection from ACS Division of Chemical Education</a></p>
  </li>
</ul>

<h2 id="books">
<a class="anchor" href="#books" aria-hidden="true"><span class="octicon octicon-link"></span></a>Books</h2>

<ul>
  <li>
    <p><a href="https://link.springer.com/book/10.1007/978-1-60761-839-3">Bajorath, 2011. Chemoinformatics and Computational Chemical Biology. Methods in Molecular Biology.</a></p>
  </li>
  <li>
    <p><a href="https://link.springer.com/book/10.1007/978-1-0716-1787-8">Heifetz, Alexander. (Ed.) (2022). “Artificial Intelligence in Drug Design.”</a></p>
  </li>
</ul>

<h2 id="best-practices">
<a class="anchor" href="#best-practices" aria-hidden="true"><span class="octicon octicon-link"></span></a>Best practices</h2>

<ul>
  <li>
<a href="https://www.nature.com/articles/s41570-022-00391-9">Bender, Andreas, et al. “Evaluation guidelines for machine learning tools in the chemical sciences.” Nature Reviews Chemistry (2022): 1-15.</a>. <a href="https://www.nature.com/articles/s41570-022-00391-9.epdf?sharing_token=c7ajAY6RtejsBwo7JKb_EdRgN0jAjWel9jnR3ZoTv0PlqwefS9tuSzOUSTdLlQspcQfdrrg6BP1js7fwhK8sXckoDuc05MIHU8Tf2mCEeVGilKBg5tdIz-htajojgKeP-9SZLLqCDAphBNd8VUtODVWYw0CXTg8CY1uUfyiB8zk%3D">Temporary SharedIt Link</a>
</li>
</ul>

<p>Nice account outlining guidelines for evaluating different AI/ML methodologies in molecular science. They propose a checklist of tests and best practices to assess the practicality and importance of different methodologies thereby providing a framework on how to evaluate plethora of ML workflows being proposed in different areas of chemical science. The basis for not overlooking the older non-ML method when evaluating the ‘new’ learning-based method, emphasis on model interpretation to translate the corrleation to chemical causality and finally</p>

<ul>
  <li><a href="https://www.nature.com/articles/s41557-021-00716-z">Artrith, Nongnuch, et al. “Best practices in machine learning for chemistry.” Nature chemistry 13.6 (2021): 505-508.</a></li>
</ul>

<p>Set of rules, considerations, and caveats to keep in mind when designing ML model for chemical science. The authors propose a checklist when evaluating ML models, while intuitive at first, when lot of the new ML papers are scanned through that lens, you can identify the shortcommings of the proposed model. This checklist is especially helpful for those entering just entering the field.</p>

<h2 id="reviews">
<a class="anchor" href="#reviews" aria-hidden="true"><span class="octicon octicon-link"></span></a>Reviews</h2>

<ul>
  <li><a href="https://pubs.rsc.org/en/content/articlelanding/2020/CS/C9CS00786E#fn1">F. Strieth-Kalthoff, F. Sandfort, M. H. S. Segler, and F. Glorius, Machine learning the ropes: principles, applications and directions in synthetic chemistry, Chem. Soc. Rev</a></li>
</ul>

<p>Pedagogical account of various machine learning techniques, models, representation schemes from perspective of synthetic chemistry. Covers different applications of machine learning in synthesis planning, property prediction, molecular design, and reactivity prediction</p>

<ul>
  <li>
    <p><a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-biodatasci-122120-124216">Rodríguez-Pérez, Raquel, Filip Miljković, and Jürgen Bajorath. “Machine Learning in Chemoinformatics and Medicinal Chemistry.” Annual review of biomedical data science 5 (2022)</a></p>
  </li>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-021-00519-x">Mariia Matveieva &amp; Pavel Polishchuk. Benchmarks for interpretation of QSAR models</a>. <a href="https://github.com/ci-lab-cz/ibenchmark">Github</a>. <a href="https://patwalters.github.io/practicalcheminformatics/jupyter/ml/interpretability/2021/06/03/interpretable.html">Patrick Walter’s blog on the paper</a>.</p>
  </li>
</ul>

<p>Paper outlining good practices for interpretating QSAR (Quantative Structure-Property Prediction) models. Good set of heuristics and comparison in the paper in terms of model interpretability. Create 6 synthetic datasets with varying complexity for QSAR tasks. The authors compare interpretability of graph-based methods to conventional QSAR methods. In regards to performance graph-based models show low interpretation compared to conventional QSAR method.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/full/10.1021/acs.accounts.0c00699">W. Patrick Walters &amp; Regina Barzilay. Applications of Deep Learning in Molecule Generation and Molecular Property Prediction</a></li>
</ul>

<p>Recent review summarising the state of the molecular property prediction and structure generation research. In spite of exciting recent advances in the modeling efforts,  there is a need to generate better (realistic)  training data, assess model prediction confidence, and metrics to quantify molecular generation performance.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/full/10.1021/acs.chemrev.1c00107?utm_source=pcm&amp;utm_medium=twitter&amp;utm_campaign=PUBS_0522_EJK_CR_chreay_CR_Most_Read&amp;src=PUBS_0522_EJK_CR_chreay_CR_Most_Read">Keith, John A., et al. “Combining machine learning and computational chemistry for predictive insights into chemical systems.” Chemical reviews 121.16 (2021): 9816-9872.</a></li>
</ul>

<p>In-depth account of the machine learning and computational methods used in material science and small molecules. Nice introduction to the mathematics and theory behind first-principles based methods.</p>

<ul>
  <li>
    <p><a href="https://chemrxiv.org/articles/preprint/Navigating_through_the_Maze_of_Homogeneous_Catalyst_Design_with_Machine_Learning/12786722/1">Navigating through the Maze of Homogeneous Catalyst Design with Machine Learning</a></p>
  </li>
  <li>
    <p><a href="https://doi.org/10.1016/j.trechm.2020.11.004">Coley, C. W. Defining and Exploring Chemical Spaces. Trends in Chemistry 2020</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.accounts.0c00699">Applications of Deep learning in molecular generation and molecular property prediction</a></p>
  </li>
  <li>
    <p><a href="https://arxiv.org/pdf/2012.05716.pdf">Utilising Graph Machine Learning within Drug Discovery and Development</a></p>
  </li>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/pii/S0169409X21001800?via%3Dihub">Machine learning directed drug formulation development</a></p>
  </li>
</ul>

<p>Review from Aspuru-Guzik and Allen’s group discussing how ML can be leveraged for various tasks in drug formulation tasks.</p>

<h2 id="industry-focused-drug-discovery-reviews">
<a class="anchor" href="#industry-focused-drug-discovery-reviews" aria-hidden="true"><span class="octicon octicon-link"></span></a>Industry-focused drug discovery reviews</h2>

<ul>
  <li>
    <p><a href="https://www.nature.com/articles/d41573-022-00025-1">Jayatunga, Madura KP, et al. “AI in small-molecule drug discovery: A coming wave.” Nat. Rev. Drug Discov 21 (2022): 175-176.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.1c01580">Abramov, Yuriy A., Guangxu Sun, and Qun Zeng. “Emerging Landscape of Computational Modeling in Pharmaceutical Development.” Journal of Chemical Information and Modeling (2022).</a></p>
  </li>
</ul>

<p>Overview of methods and scope of computational methods used in the drug development process.</p>

<ul>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c02106">Dragovich, Peter S., et al. “Small-Molecule Lead-Finding Trends across the Roche and Genentech Research Organizations.” Journal of Medicinal Chemistry (2022).</a></p>
  </li>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/pii/S1359644620305274">A. Bender and I. Cortés-Ciriano, “Artificial intelligence in drug discovery: what is realistic, what are illusions? Part 1: Ways to make an impact, and why we are not there yet,” Drug Discov. Today, vol. 26, no. 2, pp. 511–524, 2021</a></p>
  </li>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/pii/S1359644620302609">A. H. Göller et al., “Bayer’s in silico ADMET platform: a journey of machine learning over the past two decades,” Drug Discov. Today, vol. 25, no. 9, pp. 1702–1709, 2020.</a></p>
  </li>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S1740674920300032">J. Shen and C. A. Nicolaou, “Molecular property prediction: recent trends in the era of artificial intelligence,” Drug Discov. Today Technol., vol. 32–33, no. xx, pp. 29–36, 2019.</a></p>
  </li>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/pii/S1359644620305110?via%3Dihub">Mervin, L. H., Johansson, S., Semenova, E., Giblin, K. A., &amp; Engkvist, O. (2021). Uncertainty quantification in drug design. Drug discovery today, 26(2), 474-489.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/jm8000373">Congreve, Miles, et al. “Recent developments in fragment-based drug discovery.” Journal of medicinal chemistry 51.13 (2008): 3661-3680.</a></p>
  </li>
</ul>

<h2 id="special-journal-issues">
<a class="anchor" href="#special-journal-issues" aria-hidden="true"><span class="octicon octicon-link"></span></a>Special Journal Issues</h2>

<ol>
  <li>
    <p><a href="https://www.nature.com/collections/gcijejjahe">Nice collection of recent papers in Nature Communications on ML application and modeling</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/page/achre4/data-science-meets-chemistry">Data Science Meets Chemistry</a></p>
  </li>
</ol>

<p>This issue includes contributions that demonstrate the profound impact data science techniques have had in chemistry including chemical and materials synthesis, catalyst and materials design, and overhauling the models used in traditional theoretical or computational chemistry.</p>

<ol>
  <li>
    <p><a href="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01077">Journal of Medicinal Chemistry compendium of AI in Drug discovery issue</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/page/achre4/data-science-meets-chemistry">Account of Chemical Research Special Issue on advances in data-driven chemistry research</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/toc/jcisd8/62/9">Special Issue on Reaction Informatics and Chemical Space, Journal of Chemical Information and Modeling (2022)</a></p>
  </li>
</ol>

<h2 id="meeting-notes">
<a class="anchor" href="#meeting-notes" aria-hidden="true"><span class="octicon octicon-link"></span></a>Meeting notes</h2>

<ul>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/611cf1a6ac8b499b36458d19">Warr, W. (2021). National Institutes of Health (NIH) Workshop on Reaction Informatics</a></p>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/60c75883bdbb89984ea3ada5">Warr, W. (2021). Report on an NIH Workshop on Ultralarge Chemistry Databases.</a></p>
  </li>
</ul>

<h2 id="chemical-modalities">
<a class="anchor" href="#chemical-modalities" aria-hidden="true"><span class="octicon octicon-link"></span></a>Chemical modalities</h2>

<ul>
  <li><a href="https://pubs.acs.org/doi/pdf/10.1021/acsmedchemlett.9b00582">Blanco, Maria-Jesus, and Kevin M. Gardinier. “New chemical modalities and strategic thinking in early drug discovery.” ACS medicinal chemistry letters 11.3 (2020): 228-231.</a></li>
</ul>

<p>Overview of different chemical modalities currently at work to address different disease targets. The article addresses the small molecule medicinal chemists and how they can expand their outlook of small molecules to include other molecular entities when considering the angle of attack for different target engagement strategies. The authors offer a nice set of tools and thought process when selecting possible drug modalities for different target classes and what questions should be asked when zeroing in a possible mode of action.</p>

<h2 id="meta-themes-on-optimizing-small-molecules">
<a class="anchor" href="#meta-themes-on-optimizing-small-molecules" aria-hidden="true"><span class="octicon octicon-link"></span></a>Meta themes on optimizing small molecules</h2>

<ul>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0169409X02000030?via%3Dihub">Walters, W. Patrick, and Mark A. Murcko. “Prediction of ‘drug-likeness’.” Advanced drug delivery reviews 54.3 (2002): 255-271.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/jm020017n">Veber, Daniel F., et al. “Molecular properties that influence the oral bioavailability of drug candidates.” Journal of medicinal chemistry 45.12 (2002): 2615-2623.</a></p>
  </li>
</ul>

<p>Retrospective analysis on factors influencing the bioavailability of drug candidates. Authors find rotatable bonds and polar surface area or hydrogen bond count (sum of donor and accpetors) found to be important predictors of good oral bioavailability. Compounds having &lt;10 rotatable bonds and &lt;140 A (or &lt; 12 hydrogen bonds) have good chances of being orally bioavailable.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00717">DeGoey, David A., et al. “Beyond the rule of 5: lessons learned from AbbVie’s drugs and compound collection: miniperspective.” Journal of Medicinal Chemistry 61.7 (2017): 2636-2651.</a></li>
</ul>

<p>AB-MPS metric calculated using cLogD, the number of aromatic rings (nAr), and the number of rotatable bonds (nRotB) according to the formula AB-MPS = Abs(cLogD −3) + nAr + nRotB. The lower the AB-MPS score, the more likely the compound is to be absorbed, and a value of ≤14 is reported to predict a higher probability of oral absorption.</p>

<ul>
  <li>
<a href="https://www.sciencedirect.com/science/article/pii/S1367593118300176#bib0030">Poongavanam, Vasanthanathan, Bradley C. Doak, and Jan Kihlberg. “Opportunities and guidelines for discovery of orally absorbed drugs in beyond rule of 5 space.” Current Opinion in Chemical Biology 44 (2018): 23-29.</a>
Hueristics for oral bioavailability of molecules that are violating the rule of 5. MW may reach up to approximately 1000 Da provided that TPSA increases proportionally up to 250 Å2. In contrast, cLogP and HBDs must be carefully controlled at high MW. Our lack of ability to predict compound conformations and flexibility is currently a hurdle that is critical to overcome to enable further prospective design in oral bRo5 space.</li>
</ul>

<h2 id="synthesis-chemistry">
<a class="anchor" href="#synthesis-chemistry" aria-hidden="true"><span class="octicon octicon-link"></span></a>Synthesis Chemistry</h2>

<p>Catalog of recent research articles that look at synthesis chemistry from a point of view of computational workflows, how traditional synthetic chemistry methods can be combined with informatics to augment drug discovery and synthesis processes.</p>

<ul>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acscatal.2c05159#.Y6oIfVMoYEs">Ruck, Rebecca T., Neil A. Strotman, and Shane W. Krska. “The Catalysis Laboratory at Merck: 20 Years of Catalyzing Innovation.” ACS Catalysis 13 (2022): 475-503.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acs.accounts.0c00760">Dreher, Spencer D., and Shane W. Krska. “Chemistry informer libraries: Conception, early experience, and role in the future of cheminformatics.” Accounts of Chemical Research 54.7 (2021): 1586-1596.</a></p>
  </li>
</ul>

<p>Curated set of substrates to quickly assess the practicality of synthetic methods with the complete capture of success and failure, that can optimize reaction conditions with a broader scope with respect to relevant applications.</p>

<ul>
  <li>
    <p><a href="https://www.science.org/doi/pdf/10.1126/science.aat0805">Campos, Kevin R., et al. “The importance of synthetic chemistry in the pharmaceutical industry.” Science 363.6424 (2019): eaat0805.</a></p>
  </li>
  <li>
    <p><a href="https://www.science.org/doi/10.1126/science.add1383">Late-stage diversification of indole skeletons through nitrogen atom insertion</a></p>
  </li>
  <li>
    <p><a href="https://doi.org/10.1002/cbic.201800751">Lenci, Elena, and Andrea Trabocchi. “Smart Design of Small‐Molecule Libraries: When Organic Synthesis Meets Cheminformatics.” ChemBioChem 20.9 (2019): 1115-1123.</a></p>
  </li>
</ul>

<h2 id="binding-free-energetic-calculations">
<a class="anchor" href="#binding-free-energetic-calculations" aria-hidden="true"><span class="octicon octicon-link"></span></a>Binding free energetic calculations</h2>

<ul>
  <li>
    <p><a href="https://link.springer.com/article/10.1007/s10822-022-00494-x">Xu, Huafeng. “The slow but steady rise of binding free energy calculations in drug discovery.” Journal of Computer-Aided Molecular Design (2022): 1-8.</a></p>
  </li>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/pii/S2667318522000204">Thompson, James, et al. “Optimizing active learning for free energy calculations.” Artificial Intelligence in the Life Sciences 2 (2022): 100050.</a></p>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/63894b1f836ceb6cd16f7987">Pitman, Mary, et al. “To Design Scalable Free Energy Perturbation Networks, Optimal Is Not Enough.” (2022).</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.2c00879">Ganguly, Abir, et al. “AMBER Drug Discovery Boost Tools: Automated Workflow for Production Free-Energy Simulation Setup and Analysis (ProFESSA).” Journal of Chemical Information and Modeling (2022).</a></p>
  </li>
</ul>

<h2 id="cheminformatics-focus">
<a class="anchor" href="#cheminformatics-focus" aria-hidden="true"><span class="octicon octicon-link"></span></a>Cheminformatics-focus</h2>

<p>Catalog of recent reviews and manuscripts I have found useful when learning more about the state-of-the-art in Cheminformatics. I’ve tried to categorize them roughly based on their area of application:</p>

<h3 id="representation">
<a class="anchor" href="#representation" aria-hidden="true"><span class="octicon octicon-link"></span></a>Representation</h3>

<p><strong>Reviews</strong></p>

<ul>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00460-5">Representation of Molecules in NN: Molecular representation in AI-driven drug discovery: review and guide</a></p>
  </li>
  <li>
    <p><a href="https://www.nature.com/articles/s41598-018-27344-x">Screening of energetic molecules – comparing different representations</a></p>
  </li>
</ul>

<p><strong>Articles</strong></p>

<ul>
  <li>
    <p><a href="https://arxiv.org/abs/1905.13741">M. Krenn, F. Hase, A. Nigam, P. Friederich, and A. Aspuru-Guzik, “Self-Referencing Embedded Strings (SELFIES): A 100% robust molecular string representation,” Mach. Learn. Sci. Technol., pp. 1–9, 2020</a></p>
  </li>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00479-8">Could graph neural networks learn better molecular representation for drug discovery? A comparison study of descriptor-based and graph-based models</a></p>
  </li>
</ul>

<p>Comparative study of descriptor-based and graph-based models using public data set. Used descriptor-based models (XGBoost, RF, SVM, using ECFP) and compared them to graph-based models (GCN, GAT, AttentiveFP, MPNN). They show descriptor-based models outperform the graph-based models in terms of prediction accuracy and computational efficiency with SVM having best predictions. Graph-based methods are good for multi-task learning.</p>

<h3 id="predictive-modeling">
<a class="anchor" href="#predictive-modeling" aria-hidden="true"><span class="octicon octicon-link"></span></a>Predictive modeling</h3>

<ul>
  <li><a href="https://www.nature.com/articles/s42256-021-00438-4">Fang, Xiaomin, et al. “Geometry-enhanced molecular representation learning for property prediction.” Nature Machine Intelligence (2022): 1-8.</a></li>
</ul>

<p>Self-supervised learning using special type of GNN architecture (GeoGNN) that includes molecule geometric / spatial information. Geometry-enhanced molecular representation learning method (GEM). The model achieves SOTA performance on 14 of 15 public classification and regression datasets.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jcim.9b00237">Yang, K., Swanson, K., Jin, W., Coley, C., Eiden, P., Gao, H., Guzman-Perez, A., Hopper, T., Kelley, B., Mathea, M. and Palmer, A., 2019. Analyzing learned molecular representations for property prediction. Journal of chemical information and modeling, 59(8), pp.3370-3388</a></li>
</ul>

<p>Benchmark property prediction models on 19 public and 16 proprietary industrial data sets spanning a wide variety of chemical end points. Introduce a modeling framework (<strong>Chemprop</strong>) that consistently matches or outperforms models using fixed molecular descriptors as well as previous graph neural architectures on both public and proprietary data sets.</p>

<ul>
  <li><a href="https://arxiv.org/abs/2107.10402">Stuyver, T. and Coley, C.W., 2021. Quantum chemistry-augmented neural networks for reactivity prediction: Performance, generalizability and interpretability. arXiv preprint arXiv:2107.10402</a></li>
</ul>

<p>Combine structure (Graph-networks) and descriptor based features (QM-derived) to predict activation energies (E<sub>2</sub>/SN<sub>2</sub> barrier height prediction) and regioselectivity. Incorporating QM and structure leads to better overall accuracy and generalizability even in low data regions. Atom and bond level features derived using QM and used in the model generation with a smaller dataset.</p>

<h3 id="qsar-benchmarks">
<a class="anchor" href="#qsar-benchmarks" aria-hidden="true"><span class="octicon octicon-link"></span></a>QSAR benchmarks</h3>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.2c01073">Exposing the Limitations of Molecular Machine Learning with Activity Cliffs</a></li>
</ul>

<p>Account on how to treat and analyze activity cliffs in context of developing a predictive model. The authors outline best practices to probe activity cliffs. They show, using 24 DL and ML models and 30 targets, ML approaches based on molecular descriptors outperformed more complex deep learning methods. Activity cliff pairs were defined on similarity of the molecule SMILES and the bioactivity difference.  Compared to most traditional machine learning approaches, deep neural networks seem to fall short at picking up subtle structural differences (and the corresponding property change) that give rise to activity cliffs.</p>

<ul>
  <li>
    <p><a href="https://pubs.rsc.org/en/content/articlelanding/2018/sc/c7sc02664a">MoleculeNet: a benchmark for molecular machine learning (rsc.org)</a></p>
  </li>
  <li>
    <p><a href="https://pubs.rsc.org/en/content/articlelanding/2018/sc/c8sc00148k">Large-scale comparison of  machine learning methods for drug target prediction on ChEMBL - Chemical Science (RSC Publishing)</a></p>
  </li>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-017-0232-0">Beyond the hype: deep neural networks outperform established methods using a ChEMBL bioactivity benchmark set, Journal of Cheminformatics</a></p>
  </li>
</ul>

<h3 id="enumeration-of-chemical-space">
<a class="anchor" href="#enumeration-of-chemical-space" aria-hidden="true"><span class="octicon octicon-link"></span></a>Enumeration of chemical space</h3>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c01215">Subbaiah, Murugaiah AM, and Nicholas A. Meanwell. “Bioisosteres of the phenyl ring: Recent strategic applications in lead optimization and drug design.” Journal of Medicinal Chemistry 64.19 (2021): 14046-14128.</a></li>
</ul>

<p>Looks at biosteric replacements for the phenyl rings in the lead optimization phase. Phenyl rings results in improve potency but have poor solubility and lipophilicitty. Find biosteres can be used to improve them.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.1c00761">Ertl, Peter. “Magic Rings: Navigation in the Ring Chemical Space Guided by the Bioactive Rings.” Journal of Chemical Information and Modeling (2021).</a></li>
</ul>

<p>Analyze the nature of rings which appear in bioactive compounds. Ring systems are systematically extracted from one billion molecules and are analyzed to discover a structure or correlation in the bioactivity and type of rings.  No simple set of structural descriptors separating active and inactive rings could be identified, the separation is best described by a neural network model taking into account a complex combination of many substructure features.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.1c01378">Bellmann, Louis, et al. “Comparison of Combinatorial Fragment Spaces and Its Application to Ultralarge Make-on-Demand Compound Catalogs.” Journal of Chemical Information and Modeling (2022).</a></li>
</ul>

<p>Authors propose an algorithmic approach called as SpaceCompare to calculate overlap and diversity of the ultra-large combinatorial chemical libraries. The tool uses topological fragment spaces to capture the subtlties of the reaction having same product but different reactant substructures.</p>

<ul>
  <li>
<a href="https://onlinelibrary.wiley.com/doi/10.1002/minf.202100068">Zabolotna, Yuliana, et al. “NP navigator: a new look at the natural product chemical space.” Molecular informatics 40.9 (2021): 2100068.</a>.</li>
</ul>

<p>Organizing the chemical space of ChEMBL, and ZINC to compare its overlap with natural products through COCONUT. Generative Topological Mapping is used for the clustering and analysis. Helpful overview of the method with its application to drug discovery can be found <a href="https://www.sciencedirect.com/science/article/pii/S1740674920300044">here</a></p>

<ul>
  <li>
    <p><a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.6b00173">Nicolaou, Christos A., et al. “The proximal lilly collection: Mapping, exploring and exploiting feasible chemical space.” Journal of chemical information and modeling 56.7 (2016): 1253-1266.</a></p>
  </li>
  <li>
    <p><a href="https://link.springer.com/article/10.1023/A:1021271615909">Raymond, John W., and Peter Willett. “Maximum common subgraph isomorphism algorithms for the matching of chemical structures.” Journal of computer-aided molecular design 16.7 (2002): 521-533.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.8b00173">Dalke, Andrew, Jerome Hert, and Christian Kramer. “mmpdb: An open-source matched molecular pair platform for large multiproperty data sets.” Journal of chemical information and modeling 58.5 (2018): 902-910.</a></p>
  </li>
</ul>

<h3 id="explainableinterpretable-machine-learning">
<a class="anchor" href="#explainableinterpretable-machine-learning" aria-hidden="true"><span class="octicon octicon-link"></span></a>Explainable/Interpretable Machine Learning</h3>

<p><strong>Reviews/Perspectives</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c01789">Rodríguez-Pérez, Raquel, and Jürgen Bajorath. “Explainable Machine Learning for Property Predictions in Compound Optimization.” Journal of medicinal chemistry 64.24 (2021): 17744-17752</a></li>
</ul>

<p><strong>Articles</strong></p>

<ul>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/613268f0d5f0803706ba0c79">Wellawatte, Geemi P., Aditi Seshadri, and Andrew D. White. “Model agnostic generation of counterfactual explanations for molecules.” (2021).</a></p>
  </li>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-021-00519-x">Matveieva, Mariia, and Pavel Polishchuk. “Benchmarks for interpretation of QSAR models.” Journal of cheminformatics 13.1 (2021): 1-20</a>. <a href="https://patwalters.github.io/practicalcheminformatics/jupyter/ml/interpretability/2021/06/03/interpretable.html">Patrick Walter’s blog</a></p>
  </li>
</ul>

<h3 id="uncertainty-quantification">
<a class="anchor" href="#uncertainty-quantification" aria-hidden="true"><span class="octicon octicon-link"></span></a>Uncertainty quantification</h3>

<ul>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/pii/S1359644620305110">Mervin, Lewis H., et al. “Uncertainty quantification in drug design.” Drug discovery today 26.2 (2021): 474-489.</a></p>
  </li>
  <li>
    <p><a href="https://arxiv.org/pdf/2102.11439.pdf">Alan Aspuru-Guzik perspective on uncertainty and confidence</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jcim.0c00502">Uncertainty Quantification Using Neural Networks for Molecular Property Prediction. J. Chem. Inf. Model. (2020) Hirschfeld, L., Swanson, K., Yang, K., Barzilay, R. &amp; Coley, C. W.</a></p>
  </li>
</ul>

<p>Benchmark different models and uncertainty metrics for molecular property prediction.</p>

<ul>
  <li>
<a href="https://arxiv.org/abs/1910.02600">Evidential Deep learning for guided molecular property prediction and disocovery Ava Soleimany, Conor Coley, et. al.</a>. <a href="https://slideslive.com/38942396/evidential-deep-learning-for-guided-molecular-property-prediction-and-discovery">Slides</a>
</li>
</ul>

<p>Train network to output the parameters of an evidential distribution. One forward-pass to find the uncertainty as opposed to dropout or ensemble - principled incorporation of uncertainties</p>

<ul>
  <li>
    <p><a href="https://arxiv.org/pdf/2101.11588.pdf">Differentiable sampling of molecular geometries with uncertainty-based adversarial attacks</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acscentsci.0c00026">J. P. Janet, S. Ramesh, C. Duan, H. J. Kulik, ACS Cent. Sci. 2020</a></p>
  </li>
</ul>

<p>Conduct a global multi-objective optimization with expected improvement criterion. Find transition metal complex redox couples for Redox flow batteries that address stability, solubility, and redox potential metric. Use distance of a point from a training data in latent space as a metric to quantify uncertainty.</p>

<ul>
  <li><a href="https://pubs.rsc.org/en/content/articlelanding/2019/sc/c9sc02298h#!divAbstract">J. P. Janet, C. Duan, T. Yang, A. Nandy, H. J. Kulik, Chem. Sci. 2019, 10, 7913–7922</a></li>
</ul>

<p>Distance from available data in NN latent space is used as a variable for low-cost, quantitative uncertainty metric that works for both inorganic and organic chemistry. Introduce a technique to calibrate latent distances enabling conversion of distance-based metric to error estimates in units of predicted property</p>

<h3 id="active-learning">
<a class="anchor" href="#active-learning" aria-hidden="true"><span class="octicon octicon-link"></span></a>Active Learning</h3>

<p>Active learning provides strategies for efficient screening of subsets of the library. In many cases, we can identify a large portion of the most promising molecules with a fraction of the compute cost.</p>

<ul>
  <li>
    <p><a href="https://doi.org/10.1016/j.ddtec.2020.06.001">Reker, D. Practical Considerations for Active Machine Learning in Drug Discovery. Drug Discov. Today Technol. 2020</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acscentsci.0c00026">Janet, J. P., Ramesh, S., Duan, C., &amp; Kulik, H. J. (2020). Accurate multiobjective design in a space of millions of transition metal complexes with neural-network-driven efficient global optimization. ACS central science, 6(4), 513-524.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acscentsci.1c00546">A. P. Soleimany, A. Amini, S. Goldman, D. Rus, S. N. Bhatia, and C. W. Coley, “Evidential Deep Learning for Guided Molecular Property Prediction and Discovery,” ACS Cent. Sci., Jul. 2021.</a>. <a href="https://slideslive.com/38942396/evidential-deep-learning-for-guided-molecular-property-prediction-and-discovery">Slideshare</a></p>
  </li>
</ul>

<p>Train property prediction model to output a distribution statistics in single pass that describes the uncertainty. This is in contrast to using ensemble models like MC dropout. Interesting way to estimate the epistemic (due to / from model) uncertainty in the prediction. Use this approach on antibiotic search problem of Stokes et. al. Compare Chemprop and SchNet models on different tasks.</p>

<h3 id="transfer-learning">
<a class="anchor" href="#transfer-learning" aria-hidden="true"><span class="octicon octicon-link"></span></a>Transfer Learning</h3>

<p><strong>Reviews</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/pdf/10.1021/acs.jmedchem.9b02147">Cai, Chenjing, et al. “Transfer learning for drug discovery.” Journal of Medicinal Chemistry 63.16 (2020): 8683-8694.</a></li>
</ul>

<p><strong>Articles</strong></p>

<ul>
  <li><a href="https://www.nature.com/articles/s41467-019-10827-4">Approaching coupled cluster accuracy with a general-purpose neural network potential through transfer learning</a></li>
</ul>

<p>Transfer learning by training a network to DFT data and then retrain on a dataset of gold standard QM calculations (CCSD(T)/CBS) that optimally spans chemical space. The resulting potential is broadly applicable to materials science, biology, and chemistry, and billions of times faster than CCSD(T)/CBS calculations.</p>

<ul>
  <li><a href="https://iopscience.iop.org/article/10.1088/2632-2153/abae75/meta">Improving the generative performance of chemical autoencoders through transfer learning</a></li>
</ul>

<h3 id="meta-learning">
<a class="anchor" href="#meta-learning" aria-hidden="true"><span class="octicon octicon-link"></span></a>Meta Learning</h3>

<ul>
  <li><a href="https://pubs.acs.org/doi/abs/10.1021/acscentsci.6b00367">Altae-Tran, H., Ramsundar, B., Pappu, A. S., &amp; Pande, V. (2017). Low data drug discovery with one-shot learning. ACS central science, 3(4), 283-293.</a></li>
</ul>

<p>Authors demonstrate how one-shot learning can be used to signifinicantly lower the amount of data required to make predictions in drug discovery tasks. LSTM combined with GCNNs is shown to improve learning capabilities of the model. In the simplest one-shot learning formalism these continuous vectors are then fed into a simple nearest-neighbor classifier that labels new examples by distance-weighted combination of support set labels</p>

<ul>
  <li><a href="https://arxiv.org/pdf/2003.05996.pdf">Nguyen, C. Q., Kreatsoulas, C., &amp; Branson, K. M. (2020). Meta-learning GNN initializations for low-resource molecular property prediction. arXiv preprint arXiv:2003.05996.</a></li>
</ul>

<p>Use CheMBL dataset to train a gated graph neural network (GGNN) for prediction and classification tasks using meta learning protocols. Show appreciable model performance even with just approx. 256 datapoints.</p>

<h3 id="federated-learning">
<a class="anchor" href="#federated-learning" aria-hidden="true"><span class="octicon octicon-link"></span></a>Federated Learning</h3>

<ul>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-021-00576-2">Simm, Jaak, et al. “Splitting chemical structure data sets for federated privacy-preserving machine learning.” Journal of Cheminformatics 13.1 (2021): 1-14.</a></p>
  </li>
  <li>
    <p><a href="https://www.melloddy.eu/objectives">Melloddy consortium</a></p>
  </li>
</ul>

<p>Consortia comprising of leading resarch labs and companies working on decentralized datasets and predictive modeling of biochemical and cellular activity.</p>

<h3 id="generative-models">
<a class="anchor" href="#generative-models" aria-hidden="true"><span class="octicon octicon-link"></span></a>Generative models</h3>

<p><strong>Reviews</strong></p>

<ul>
  <li><a href="https://www.nature.com/articles/s41587-020-0418-2">Walters, W. Patrick, and Mark Murcko. “Assessing the impact of generative AI on medicinal chemistry.” Nature biotechnology 38.2 (2020): 143-145.</a></li>
</ul>

<p>Correspondence on assessing the impact of AI on medicinal chemistry. It is a well written account on practical implication of generative design on pharmaceutical research.They outline two recent cases of ‘success’ of AI generative design in drug discovery and give more context and propose best practices for furthering the development of algorithms and drug discovery pipelines.</p>

<ul>
  <li>
    <p><a href="https://pubmed.ncbi.nlm.nih.gov/33562347/">Mouchlis VD, Afantitis A, Serra A, et al. Advances in de Novo Drug Design: From Conventional to Machine Learning Methods. Int J Mol Sci. 2021;22(4):1676. Published 2021 Feb 7. doi:10.3390/ijms22041676</a></p>
  </li>
  <li>
    <p><a href="https://science.sciencemag.org/content/361/6400/360">B. Sanchez-Lengeling and A. Aspuru-Guzik, “Inverse molecular design using machine learning: Generative models for matter engineering,” Science (80)., vol. 361, no. 6400, pp. 360–365, Jul. 2018</a></p>
  </li>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/pii/S1359644621002531#f0010">Meyers, Joshua, Benedek Fabian, and Nathan Brown. “De novo molecular design and generative models.” Drug Discovery Today 26.11 (2021): 2707-2715.</a></p>
  </li>
</ul>

<p>Very nice review of different atom-based, reaction-based, and fragment-based generative design workflows proposed by the community.</p>

<p><strong>Benchmarks</strong></p>

<ul>
  <li>
<a href="https://arxiv.org/abs/2112.03041">Flam-Shepherd, Daniel, Kevin Zhu, and Alán Aspuru-Guzik. “Keeping it Simple: Language Models can learn Complex Molecular Distributions.” arXiv preprint arXiv:2112.03041 (2021).</a>. <a href="https://www.nature.com/articles/s41467-022-30839-x">Nature Comms Link</a>
</li>
</ul>

<p>Test SOTA language models and representation performance against graph-based methods (CGVAE, JTVAE) for ‘challenging’ generative modeling tasks - generate a molecule - property distribution as a function of synthetic feasiblity. Graph models faced chanllenge in generating large molcules (&gt; 100 HAs). Selfies provided advantage here. All of the models seem to generate novel molecules - how practical each of these novel molecules are is yet an open question.</p>

<ul>
  <li>
<a href="https://arxiv.org/abs/1811.12823">MOSES - Benchmarking platform for generative models</a>.</li>
</ul>

<p>Propose a platform to deploy and compare state-of-the-art generative models for exploring molecular space on same dataset. In addition the authors also propose list of metrics  to evaluate the quality and diversity of the generated structures.</p>

<ul>
  <li>
<a href="https://www.benevolent.com/guacamol">GuacaMol: Benchmarking models for De Novo Molecular Design</a>. <a href="https://www.benevolent.com/research/guacamol-benchmarking-models-for-de-novo-molecular-design">Blogpost</a>
</li>
</ul>

<p>Evaluation framework from BenevolentAI to compare different de-novo design models.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.0c01328">J. Zhang, R. Mercado, O. Engkvist, and H. Chen, “Comparative Study of Deep Generative Models on Chemical Space Coverage,” J. Chem. Inf. Model., vol. 61, no. 6, pp. 2572–2581, Jun. 2021.</a></li>
</ul>

<p>Interesting analysis from team at AstraZeneca R&amp;D. They look at the chemical space coverage accounted by the SOTA generative models. Proposes a metric for evaluating space coverage, and thereby comparing different SOTA models, using a reference data (GDB-13 in this case). The new metric computes how much of the GDB-13 dataset can be recovered by a model that is trained on small GDB subset.  Generative models were trained on same 1M data points and 1B molecules were then sampled from each model. It was seen that at most 39% of the molecules in the parent dataset were sampled / generated by the model. Most models sampled the same compounds atleast twice. It was observed that graph-based model sampled much diverse molecules than string-based methods. Besides, the coverage of GAN-based models was worse compared to Language and Graph models.</p>

<ul>
  <li><a href="https://doi.org/10.1021/acs.jcim.0c00174">Gao, W.; Coley, C. W. The Synthesizability of Molecules Proposed by Generative Models. J. Chem. Inf. Model. 2020</a></li>
</ul>

<p>This paper looks at different ways of integrating synthesizability criteria into generative models.</p>

<ul>
  <li><a href="https://chemrxiv.org/engage/chemrxiv/article-details/614757fa6fc3a870f6a4bbe2">Comparative analysis of graph traversal schemes for GraphINVENT</a></li>
</ul>

<p>Bechmark work from AstraZeneca/MIT AI team to document different graph architecture schemes and algorithms for generative models.</p>

<p><strong>Language models:</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acscentsci.7b00572">R. Gómez-Bombarelli et al., “Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules,” ACS Cent. Sci., vol. 4, no. 2, pp. 268–276, 2018</a></li>
</ul>

<p>One of the first implementation of a variation auto-encoder for molecule generation</p>

<ul>
  <li>
    <p><a href="https://s3-eu-west-1.amazonaws.com/itempdf74155353254prod/7977131/Penalized_Variational_Autoencoder_for_Molecular_Design_v2.pdf">Penalized Variational Autoencoder</a></p>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/articles/preprint/Beyond_Generative_Models_Superfast_Traversal_Optimization_Novelty_Exploration_and_Discovery_STONED_Algorithm_for_Molecules_using_SELFIES/13383266">SELFIES and generative models using STONED</a></p>
  </li>
</ul>

<p>Representation using SELFIES proposed to make it much more powerful</p>

<ul>
  <li>
    <p><a href="https://arxiv.org/pdf/2102.00700.pdf">Reproducibility study of the STONED work from Jablonka et. al.</a></p>
  </li>
  <li>
    <p><a href="https://s3-eu-west-1.amazonaws.com/itempdf74155353254prod/10119299/Generating_Customized_Compound_Libraries_for_Drug_Discovery_with_Machine_Intelligence_v1.pdf">LSTM based (RNN) approaches to small molecule generation</a>. <a href="https://github.com/ETHmodlab/BIMODAL">Github</a></p>
  </li>
  <li>
    <p><a href="https://arxiv.org/abs/2010.09885">Chithrananda, S.; Grand, G.; Ramsundar, B. ChemBERTa: Large-Scale Self-Supervised Pretraining for Molecular Property Prediction. arXiv [cs.LG], 2020</a>.</p>
  </li>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00441-8#availability-of-data-and-materials">SMILES-based deep generative scaffold decorator for de-novo drug design</a>. <a href="https://github.com/undeadpixel/reinvent-randomized">Github</a></p>
  </li>
</ul>

<p>SMILES-based language model that generates molecules from scaffolds and can be trained from any arbitrary molecular set. Uses randomized SMILES to improve final prediction validity.</p>

<ul>
  <li><a href="https://chemrxiv.org/engage/chemrxiv/article-details/60c7591a9abda2847ff8ea1f">Iovanac, Nicolae C., Robert MacKnight, and Brett Savoie. “Actively Searching: Inverse Design of Novel Molecules with Simultaneously Optimized Properties.” ChemRxiv (2021)</a></li>
</ul>

<p>Using quantum chemistry attributes calculated on-the-fly as scoring functions for sampling the generative model chemical space. Active learning strategy is deployed to explore the area of space where the properties of the molecules are unknown.</p>

<p><strong>Graph-based</strong></p>

<ul>
  <li><a href="https://arxiv.org/abs/2202.00658?">Flam-Shepherd, Daniel, Alexander Zhigalin, and Alán Aspuru-Guzik. “Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning.” arXiv preprint arXiv:2202.00658 (2022)</a></li>
</ul>

<p>Reinforcement learning-based generative model whici is an update on point cloud approach by the same group to now incorporate ‘grammar’ for building molecules in form of functional groups in 3D space.</p>

<ul>
  <li><a href="https://arxiv.org/abs/1802.04364">W. Jin, R. Barzilay, and T. Jaakkola, “Junction tree variational autoencoder for molecular graph generation,” 35th Int. Conf. Mach. Learn. ICML 2018, vol. 5, pp. 3632–3648, 2018</a></li>
</ul>

<p>Junction tree based decoding. Define a grammar for the small molecule and find sub-units based on that grammar to construct a molecule. The molecule is generated in two-steps: first being generating the scaffold or backbone of the molelcule, then the nodes  are added with molecular substructure as identified from the ‘molecular grammar’.</p>

<ul>
  <li><a href="https://iopscience.iop.org/article/10.1088/2632-2153/abf5b7/pdf">MPGVAE: Message passing graph networks for molecular generation, Daniel Flam-Shepherd et al 2021 Mach. Learn.: Sci. Technol.</a></li>
</ul>

<p>Introduce a graph generation model by building a Message Passing Neural Network (MPNNs) into the encoder and decoder of a VAE (MPGVAE).</p>

<ul>
  <li><a href="https://arxiv.org/pdf/2105.07246.pdf">ConfVAE: End-to-end framework for molecular conformation generation via bilevel programming</a></li>
</ul>

<p>Algorithm to predict 3D conforms from molecular graphs.</p>

<ul>
  <li>
<a href="https://iopscience.iop.org/article/10.1088/2632-2153/abcf91/meta">GraphINVENT: R. Mercado, T. Rastemo, E. Lindelöf, G. Klambauer and O. Engkvist, “Graph networks for molecular design,” Mach. Learn. Sci. Technol., vol. 2, no. 2, p. 25023, 2021</a>. <a href="https://github.com/MolecularAI/GraphINVENT">Github</a>. <a href="https://www.cheminformania.com/using-graphinvent-to-generate-novel-drd2-actives/">Blogpost</a>
</li>
</ul>

<p>GraphINVENT uses a tiered deep neural network architecture to probabilistically generate new molecules a single bond at a time.</p>

<ul>
  <li><a href="https://chemrxiv.org/engage/chemrxiv/article-details/60fc07bd171fc7a0adb87039">RL-GraphINVENT: Reinforcement learning-based variant of the above code.</a></li>
</ul>

<p><strong>GANs</strong></p>

<ul>
  <li><a href="https://arxiv.org/abs/1805.11973">MolGAN: An implicit generative model for small molecular graphs, N. De Cao and T. Kipf, 2018</a></li>
</ul>

<p>Generative adversarial network for finding small molecules using graph networks, quite interesting. Avoids issues arising from node ordering that are associated with likelihood based methods by using an adversarial loss instead (GAN)</p>

<ul>
  <li><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0397-9">LatentGAN: A de novo molecular generation method using latent vector based generative adversarial network</a></li>
</ul>

<p>Molecular generation strategy is described which combines an autoencoder and a GAN. Generator and discriminator network do not use SMILES strings as input, but instead n-dimensional vectors derived from the code-layer of an autoencoder trained as a SMILES heteroencoder that way syntax issues are expected to be addressed.</p>

<p><strong>Scaffold-retained</strong></p>

<ul>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.1c01130">Kaitoh, Kazuma, and Yoshihiro Yamanishi. “Scaffold-Retained Structure Generator to Exhaustively Create Molecules in an Arbitrary Chemical Space.” Journal of Chemical Information and Modeling (2022).</a></p>
  </li>
  <li>
    <p><a href="https://arxiv.org/pdf/2103.03864.pdf">Maziarz, Krzysztof, et al. “Learning to extend molecular scaffolds with structural motifs.” arXiv preprint arXiv:2103.03864 (2021).</a></p>
  </li>
</ul>

<p>Team at Novartis and Microsoft propose MoLeR, graph based model to generate molecule using scaffold as a seed. Scaffold based SAR speed up shown.</p>

<p><strong>Reaction tranformation-based</strong></p>

<p>Here the idea is to constraint the molecules generated by the transformations amenable to a particular platform, like automated synthesis workflow.</p>

<ul>
  <li>
    <p><a href="https://onlinelibrary.wiley.com/doi/10.1002/advs.202206674?utm_source=pocket_saves">Seo, Seonghwan, Jaechang Lim, and Woo Youn Kim. “Molecular Generative Model via Retrosynthetically Prepared Chemical Building Block Assembly.” Advanced Science (2023): 2206674.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.1c00469">Fialková, Vendy, et al. “LibINVENT: reaction-based generative scaffold decoration for in silico library design.” Journal of Chemical Information and Modeling 62.9 (2021): 2046-2063.</a></p>
  </li>
  <li>
    <p><a href="https://arxiv.org/pdf/2106.03394.pdf">Nguyen, Dai Hai, and Koji Tsuda. “A generative model for molecule generation based on chemical reaction trees.” arXiv preprint arXiv:2106.03394 (2021).</a></p>
  </li>
</ul>

<p>Authors propose a generative model to generate molecules via multi-step chemical reaction trees, each campaign first generates a reaction-tree with template transformations as breaking points.</p>

<p><strong>3D conformations-aware</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.1c01535">Bolcato, Giovanni, Esther Heid, and Jonas Boström. “On the Value of Using 3D Shape and Electrostatic Similarities in Deep Generative Methods.” Journal of chemical information and modeling 62.6 (2022): 1388-1398.</a></li>
</ul>

<p>Extension to the fragment-based generative design model (DeepFMPO) using reinforcement learning now incorporating 3D electrostatic similarity in the analysis. Ability to replace fragment with similar 3D shape and electrostatics. ESP_sim for comparison of electrostatic potential and molecule shape is used for this purpose. The authors find scaffold-hopping bioisoteres for CDK2.</p>

<ul>
  <li><a href="https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc02436a#!">Imrie, Fergus, et al. “Deep generative design with 3D pharmacophoric constraints.” Chemical science 12.43 (2021): 14577-14589.</a></li>
</ul>

<p>Method that combines GNNs with CNNs to incorporate 3D pharmacophoric constraints into molecular generation.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.9b01120">Imrie, Fergus, et al. “Deep generative models for 3D linker design.” Journal of chemical information and modeling 60.4 (2020): 1983-1995.</a></li>
</ul>

<p>Interesting work on designing linkers using conformation aware generative design algorithm. Think of it like fragment-growing.</p>

<p><strong>Protein-ligand interactions aware</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c00177">Zhang, Jie, and Hongming Chen. “De novo molecule design using molecular generative models constrained by ligand–protein interactions.” Journal of Chemical Information and Modeling 62.14 (2022): 3291-3306.</a></li>
</ul>

<h3 id="computer-aided-synthesis-planning-casp">
<a class="anchor" href="#computer-aided-synthesis-planning-casp" aria-hidden="true"><span class="octicon octicon-link"></span></a>Computer Aided Synthesis Planning (CASP)</h3>

<p><strong>Reviews:</strong></p>

<ul>
  <li><a href="https://pubs.rsc.org/en/content/articlehtml/2021/xx/d0re00340a?casa_token=XlT3Q35wF_QAAAAA:Rcge3W9WUQher8k9zVMPBUQJsCeOskpUZOvQShiCcVZw127BHRFMRrZL0X6Upa9eHD-KMYPBNTaPww">Thakkar, Amol, et al. “Artificial intelligence and automation in computer aided synthesis planning.” Reaction chemistry &amp; engineering 6.1 (2021): 27-51.</a></li>
</ul>

<p>Perspective on the current SOTA of synthesis planning, automation, and reaction optimization in drug discovery and development phases using AI and ML.</p>

<ul>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0959943621002959?dgcid=author">Madzhidov, T. I., et al. (2021). “Machine learning modelling of chemical reaction characteristics: yesterday, today, tomorrow.” Mendeleev Communications 31(6): 769-780.</a></p>
  </li>
  <li>
    <p><a href="https://www.nature.com/articles/s41570-021-00260-x">Jorner, K., et al. (2021). “Organic reactivity from mechanism to machine learning.” Nature Reviews Chemistry 5(4): 240-255.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/pdf/10.1021/acs.jmedchem.9b02120">Struble, T. J., et al. (2020). “Current and Future Roles of Artificial Intelligence in Medicinal Chemistry Synthesis.” J Med Chem 63(16): 8667-8682</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/full/10.1021/acs.accounts.0c00770">Zuranski, Andrzej M., et al. “Predicting reaction yields via supervised learning.” Accounts of chemical research 54.8 (2021): 1856-1865.</a></p>
  </li>
</ul>

<p>Perspective on ML for organic chemistry reactivity prediction. Group uses DFT-derived physical features of the reaction molecules and conditions for representation. Small data set plus HTE experimentation dataset for yield estimation.</p>

<ul>
  <li><a href="https://arxiv.org/pdf/1906.10223.pdf">The Exploration of Chemical Reaction Networks</a></li>
</ul>

<p>Perspective article summarising their position on the current state of research and future considerations on developing better reaction network models. Break down the analysis of reaction networks as into 3 classes (1) Front Open End: exploration of products from reactants (2) Backward Open Start: Know the product and explore potential reactants (3) Start to End: Product and reactant known, explore the likely intermediates.</p>

<p>Nice summary of potential challenges in the field:</p>

<ul>
  <li>Validating exploration algorithms on a consistent set of reaction system.</li>
  <li>Need to generate a comparative metric to benchmark different algorithms.</li>
  <li>
    <p>Considering effect of solvents and/or protein embeddings in the analysis</p>

    <ul>
      <li>Previous review article by same group: <a href="https://pubs.acs.org/doi/pdf/10.1021/acs.jpca.8b10007">Exploration of Reaction Pathways and Chemical Transformation Networks</a>
</li>
    </ul>
  </li>
</ul>

<p>Technical details of various algorithms being implemented for reaction mechanism discovery at the time of writing the review.</p>

<p><strong>Best practices</strong></p>

<ul>
  <li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/minf.202100119?casa_token=SczoexCIDDUAAAAA:3M-uhrRFMaNRnydOQb-NBhJ1VvocjwZ1_ll--OGP1QwJ-X5Amwt24M586NXxbbgazYn0t-NBAmxS1oFk">Gimadiev, T. R., Lin, A., Afonina, V. A., Batyrshin, D., Nugmanov, R. I., Akhmetshin, T., … &amp; Varnek, A. (2021). Reaction Data Curation I: Chemical Structures and Transformations Standardization. Molecular Informatics, 2100119.</a></li>
</ul>

<p>Article from Varnek group on best practices on processing data for reaction informatics.</p>

<p><strong>Benchmarking</strong></p>

<ul>
  <li>
<a href="https://chemrxiv.org/engage/chemrxiv/article-details/621c86f3c3e9da4f737b0047">Genheden S, Bjerrum E. PaRoutes: a framework for benchmarking retrosynthesis route predictions. ChemRxiv. Cambridge: Cambridge Open Engage; 2022</a>. <a href="https://github.com/MolecularAI/PaRoutes">Github</a>
</li>
</ul>

<p>Benchmarking framework for comparing different multi-step retrosynthesis methods from researchers at AstraZeneca R&amp;D. Provides 10k synthetic routes which can be used as a validation set for different methodologies, providing a platform for systematic comparison of different methods being proposed in the community.</p>

<p><strong>Classifying chemical reactions:</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/ci5006614">Schneider, N., et al. (2015). “Development of a Novel Fingerprint for Chemical Reactions and Its Application to Large-Scale Reaction Classification and Similarity.” Journal of Chemical Information and Modeling 55(1): 39-53.</a></li>
</ul>

<p>Using scrapped US Patent data to classify chemical reactions and deploy various fingerprints and ML models for classification.</p>

<ul>
  <li>
<a href="https://www.nature.com/articles/s42256-020-00284-w">Schwaller, Philippe, et al. “Mapping the space of chemical reactions using attention-based neural networks.” Nature Machine Intelligence 3.2 (2021): 144-152.</a>. <a href="https://github.com/rxn4chemistry/rxnfp">rxnfp - Github</a>. <a href="https://arxiv.org/abs/2012.06051">Preprint</a>. <a href="https://cen.acs.org/physical-chemistry/computational-chemistry/Mapping-reaction-space-machine-learning/99/i5?utm_source=LatestNews&amp;utm_medium=LatestNews&amp;utm_campaign=CENRSS">News Article</a>.</li>
</ul>

<p>Transformer-based model for reaction classification. Compared it with BERT. Besides classification, the work also formalizes the reaction fingerprint generation using the learned representations. The reaction fingerprints are visualized using TMAPS.</p>

<ul>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/60e358fb379e8d3ba9f92d15">Probst, Daniel, Philippe Schwaller, and Jean-Louis Reymond. “Reaction Classification and Yield Prediction Using the Differential Reaction Fingerprint DRFP.” ChemRxiv (2021)</a></p>
  </li>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00476-x">Delannée, V., Nicklaus, M.C. ReactionCode: format for reaction searching, analysis, classification, transform, and encoding/decoding. J Cheminform 12, 72 (2020)</a></p>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/6112ac487117507542e68bef">Heid, E; Green, W; Machine learning of reaction properties via learned representations of the condensed graph of reaction. ChemRxiv (2021)</a></p>
  </li>
</ul>

<p>Reaction classifiction prediction using atom-mapped reaction that are used to generate condensed reaction graphs and passed through a GCN-variant as implemented in chemprop.</p>

<p><strong>Atom mapping:</strong></p>

<ul>
  <li><a href="https://onlinelibrary.wiley.com/doi/10.1002/minf.202100138?af=R">Lin, A., et al. (2021). “Atom-to-atom Mapping: A Benchmarking Study of Popular Mapping Algorithms and Consensus Strategies.”</a></li>
</ul>

<p>Comparative analysis of different atom-mapping schemes for generating atom-mapped reaction features. Comments on the state of the art methods and their performance on a curated reaction database.</p>

<ul>
  <li>
<a href="https://advances.sciencemag.org/content/7/15/eabe4166">Extraction of organic chemistry grammar from unsupervised learning of chemical reactions</a>. <a href="https://github.com/rxn4chemistry/rxnmapper">RXMapper</a>
</li>
</ul>

<p>Data-driven atom mapping schemes which uses transformers for learning the context of the chemical reaction. Researchers at IBM trained a flavor of language model based on Transformer architecture and used it to find reaction centers and maps atoms. Shown to be robust compared to other SOTA methods.</p>

<ul>
  <li><a href="https://www.nature.com/articles/s41467-019-09440-2">Automatic mapping of atoms across both simple and complex chemical reactions</a></li>
</ul>

<p><strong>Predicting reaction outcomes:</strong></p>

<ul>
  <li>
    <p><a href="https://arxiv.org/abs/1709.04555">Jin, Wengong, et al. “Predicting organic reaction outcomes with weisfeiler-lehman network.” Advances in neural information processing systems 30 (2017).</a></p>
  </li>
  <li>
    <p><a href="https://pubs.rsc.org/en/content/articlepdf/2019/sc/c8sc04228d">C. W. Coley et al., “A graph-convolutional neural network model for the prediction of chemical reactivity,” Chem. Sci., vol. 10, no. 2, pp. 370–377, 2019.</a></p>
  </li>
</ul>

<p>Template-free prediction of organic reaction outcomes using graph convolutional neural networks</p>

<ul>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acscentsci.7b00064?ref=acsciiVIdeepchemistry">Prediction of Organic Reaction Outcomes Using Machine Learning, ACS Cent. Sci. 2017</a></p>
  </li>
  <li>
    <p><a href="https://pubs.rsc.org/en/content/articlehtml/2021/sc/d0sc04823b">Guan, Y., et al. (2021). “Regio-selectivity prediction with a machine-learned reaction representation and on-the-fly quantum mechanical descriptors.” Chemical Science 12(6): 2198-2208</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acscentsci.9b00576">Schwaller, P., et al. (2019). “Molecular Transformer: A Model for Uncertainty-Calibrated Chemical Reaction Prediction.” ACS Central Science 5(9): 1572-1583.</a></p>
  </li>
  <li>
    <p><a href="https://iopscience.iop.org/article/10.1088/2632-2153/abc81d">Schwaller, P., et al. (2021). “Prediction of chemical reaction yields using deep learning.” Machine Learning: Science and Technology 2(1)</a></p>
  </li>
</ul>

<p><strong>Retrosynthetic routes:</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.1c00754">Zabolotna, Y., et al. (2021). “SynthI: A New Open-Source Tool for Synthon-Based Library Design.” Journal of Chemical Information and Modeling.</a></li>
</ul>

<p>Interesting work on de-novo design of molecules wherein, the molecules being created are made up from the fragments that is known to exist and are available to the user. New molecules are generated based on the fragmented (synthons) made available in the dataset.</p>

<ul>
  <li>
    <p><a href="https://pubs.rsc.org/en/content/articlelanding/2020/sc/c9sc05704h">Schwaller, P., et al. (2020). “Predicting retrosynthetic pathways using transformer-based models and a hyper-graph exploration strategy.” Chemical Science 11(12): 3316-3325.</a></p>
  </li>
  <li>
    <p><a href="https://www.nature.com/articles/s41586-020-2855-y">Computational planning of the synthesis of complex natural products</a></p>
  </li>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0323-6">Watson, I. A., et al. (2019). “A retrosynthetic analysis algorithm implementation.” J Cheminform 11(1)</a></p>
  </li>
  <li>
    <p><a href="https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/chem.201605499">Segler, Marwin HS, and Mark P. Waller. “Neural‐symbolic machine learning for retrosynthesis and reaction prediction.” Chemistry–A European Journal 23.25 (2017): 5966-5971.</a></p>
  </li>
</ul>

<p>Hybrid neural-symbolic approach for both retrosynthesis and reaction prediction that can be trained with large reaction sets from databases. Template extraction from known reaction datasets to classify new reaction to known reaction classes.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.0c00403">Fortunato, Michael E., et al. “Data augmentation and pretraining for template-based retrosynthetic prediction in computer-aided synthesis planning.” Journal of chemical information and modeling 60.7 (2020): 3398-3407.</a></li>
</ul>

<p>In template-based retrosynthesis predictions, templates with few examples are excluded from training. This works talks on methods to augment the current set of data to account for the cases where examples for training are few.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.1c01065">Seidl, Philipp, et al. “Improving Few-and Zero-Shot Reaction Template Prediction Using Modern Hopfield Networks.” Journal of chemical information and modeling 62.9 (2022): 2111-2120.</a></li>
</ul>

<p>Introduce a template-based single-step retrosynthesis model based on Modern Hopfield
Networks, which learn an encoding of both molecules and reaction templates in order to
predict the relevance of templates for a given molecule. The model does not consider templates as distinct categories, but can leverage structural information about the template. The retrieval approach enables generalization across templates, which makes zero-shot learning possible and improves few-shot learning. On the single-step retrosynthesis benchmark USPTO-50k, the MHN model reaction reaches the state-of-the-art at top-k accuracy for k ≥ 3.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c00321">Tu, Zhengkai, and Connor W. Coley. “Permutation invariant graph-to-sequence model for template-free retrosynthesis and reaction prediction.” Journal of Chemical Information and Modeling (2021).</a></li>
</ul>

<p>Graph2SMILES, a template-free retrosynthesis model to predict reaction outcomes and retrosynthesis routes. This model eliminates the need for any input-side SMILES augmentation, while achieving noticeable improvements over Transformer baselines (especially for top-1 accuracy).</p>

<p><strong>Generate reaction networks:</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.0c01480">M. Liu et al., “Reaction Mechanism Generator v3.0: Advances in Automatic Mechanism Generation,” J. Chem. Inf. Model., May 2021</a></li>
</ul>

<p>Newest version of RMG (v3) is updated to Python v3. It has ability to generate heterogeneous catalyst models, uncertainty analysis to conduct first order sensitivity analysis. RMG dataset for the thermochemical and kinetic parameters have been expanded.</p>

<ul>
  <li><a href="https://s3-eu-west-1.amazonaws.com/itempdf74155353254prod/13076087/More_and_Faster__Simultaneously_Improving_Reaction_Coverage_and_Computational_Cost_in_Automated_Reaction_Prediction_Task_v1.pdf">More and Faster: Simultaneously Improving Reaction Coverage and Computational Cost in Automated Reaction Prediction Tasks</a></li>
</ul>

<p>Presents an algorithmic improvement to the reaction network prediction task through their YARP (Yet Another Reaction Program) methodology. Shown to reduce computational cost of optimization while improving the diversity of identified products and reaction pathways.</p>

<ul>
  <li>
<a href="https://pubs.acs.org/doi/abs/10.1021/acscentsci.9b00576">Molecular Transformer: A Model for Uncertainty-Calibrated Chemical Reaction Prediction</a>
    <ul>
      <li>Follow-up: <a href="https://www.nature.com/articles/s41467-021-21895-w">Quantitative interpretation explains machine learning models for chemical reaction prediction and uncovers bias</a>
</li>
    </ul>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/articles/preprint/Automatic_discovery_of_chemical_reactions_using_imposed_activation/13008500/1">Automatic discovery of chemical reactions using imposed activation</a></p>
  </li>
  <li><a href="https://www.nature.com/articles/s41467-020-19267-x">Machine learning in chemical reaction space</a></li>
</ul>

<p>Look at exploration of reaction space rather than compound space. SOAP kernel for representing the moelcules. Estimate atomization energy for the molecules using ML. Calculate the d(AE) for different ML-estimated AEs. Reaction energies (RE) are estimated and uncertainty propogation is used to estimate the errors. Uncorrelated constant error propogation. 30,000 bond breaking reaction steps Rad-6-RE network used. RE prediction is not as good as AE.</p>

<p><strong>Estimate molecular synthesizability</strong></p>

<p>The idea of estimating whether a molecule is ‘synthesizable’ can be thought of from two areas:</p>
<ol>
  <li>Complexity based - compare the fragments in the molecule to the known fragments in the chemical space</li>
  <li>Full retrosynthesis based - entire route is considered for molecule generation. Reactant complexity drives route complexity.</li>
</ol>

<ul>
  <li>
<a href="https://jcheminf.biomedcentral.com/articles/10.1186/1758-2946-1-8">Ertl, Peter, and Ansgar Schuffenhauer. “Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions.” Journal of cheminformatics 1.1 (2009): 1-11.</a>. <a href="https://github.com/rdkit/rdkit/tree/master/Contrib/SA_Score">RDkit implementation</a>
</li>
</ul>

<p>Synthetic Accessbility  score (SA_Score) is a popular heuristic score for quantifying synthesizability. It computes a score using a fragment-contribution approach, where rarer fragments (as judged by their abundance in the PubChem database of 1mil representative cmpds) are taken as an indication of lower synthesizability.</p>

<ul>
  <li>
<a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.7b00622">Coley, Connor W., et al. “SCScore: synthetic complexity learned from a reaction corpus.” Journal of chemical information and modeling 58.2 (2018): 252-261.</a>. <a href="https://github.com/deepchem/deepchem/blob/master/examples/tutorials/Synthetic_Feasibility_Scoring.ipynb">DeepChem implementation</a>
</li>
</ul>

<p>SCScore is a learned synthetic complexity score computed as a neural network model trained on reaction data from the Reaxys database. It was designed with synthesis planning in mind to operate on molecules resembling not just drug-like products but intermediates and simpler building blocks as well.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.1c01476">Liu, Cheng-Hao, et al. “RetroGNN: Fast Estimation of Synthesizability for Virtual Screening and De Novo Design by Learning from Slow Retrosynthesis Software.” Journal of Chemical Information and Modeling 62.10 (2022): 2293-2300.</a></li>
</ul>

<p>RetroGNN is a graph neural network based model to predict outcome of a synthesis planner given the target molecule. Shown to better perform than SAScore. Code is yet to be released.</p>

<h3 id="data-driven-chemistry-modeling-and-reaction-optimization">
<a class="anchor" href="#data-driven-chemistry-modeling-and-reaction-optimization" aria-hidden="true"><span class="octicon octicon-link"></span></a>Data-driven chemistry modeling and reaction optimization</h3>

<p><strong>Review</strong></p>

<ul>
  <li><a href="https://pubs.acs.org/doi/full/10.1021/acscentsci.1c00535">Williams, Wendy L., et al. “The evolution of data-driven modeling in organic chemistry.” ACS central science 7.10 (2021): 1622-1637.</a></li>
</ul>

<p><strong>Articles</strong></p>

<ul>
  <li>
<a href="https://www.nature.com/articles/s41586-021-03213-y">B. J. Shields et al., “Bayesian reaction optimization as a tool for chemical synthesis,” Nature, vol. 590, no. June 2020, p. 89, 2021</a>. <a href="https://github.com/b-shields/edbo">Github</a>
</li>
</ul>

<p>Experimental design using Bayesian Optimization. Look at 3 rxn class with multiple reaction parameters - temp solvent ligand. Algorithm identifies the optimal conditions. Variables looked into: ligands, bases, solvents, temperatures, concentrations. Algorithm arrived at 99% yields consistently - which was possible by using unusual ligand not known to work well (cognitive bias).</p>

<ul>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/62f6966269f3a5df46b5584b">Torres, Jose Garrido, et al. “A Multi-Objective Active Learning Platform and Web App for Reaction Optimization.” (2022).</a></p>
  </li>
  <li>
    <p><a href="https://arxiv.org/abs/2203.17241">Hickman, Riley J., et al. “Bayesian optimization with known experimental and design constraints for chemistry applications.” arXiv preprint arXiv:2203.17241 (2022).</a></p>
  </li>
  <li>
    <p><a href="https://aip.scitation.org/doi/abs/10.1063/5.0048164">Häse, Florian, et al. “Gryffin: An algorithm for Bayesian optimization of categorical variables informed by expert knowledge.” Applied Physics Reviews 8.3 (2021): 031406.</a></p>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/62b2dc4b7da6ce2ddb1b3264">Dotson, Jordan, et al. “Data-driven multi-objective optimization tactics for catalytic asymmetric reactions.” (2022).</a></p>
  </li>
</ul>

<p>Multi-objective optimization of catalytic reactions that employ chiral bisphosphine ligands. Optimization of 2 sequential reactions in asymmetric synthesis of API. Classification method identify active catalysts – 5% yield (user provided) cutoff for binary classification. Linear regression to model reaction selectivity. DFT-derived descriptor dataset of &gt;550 bisphosphine ligands. Develop an interpretable chemical space mapping tool using PCA. Look at the domain of applicability with the euclidean distance in chemical space.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/10.1021/acscatal.2c02807#.YvpbIMo0Ovw.linkedin">Zhang, Ying, et al. “Descriptor-Free Design of Multicomponent Catalysts.” ACS Catalysis 12 (2022): 10562-10571.</a></li>
</ul>

<p>Bayesian optimization (BO) to improve the experimental measured activity as a direct function of compositional variables without educating physical knowledge to the machine. We applied BO in screening spinel Cr<sub>a</sub>Mn<sub>b</sub>Fe<sub>c</sub>Co<sub>d</sub>Ni<sub>e</sub>Cu<sub>f</sub>Zn<sub>3–a–b–c–d–e–f</sub>O<sub>4</sub> for the decomposition of nitric oxide into environmentally friendly nitrogen.</p>

<p><strong>Databases</strong></p>

<ul>
  <li>
    <p><a href="https://pubs.acs.org/doi/full/10.1021/jacs.1c09820?utm_source=pocket_mylist">Kearnes, S. M., et al. (2021). “The Open Reaction Database.” Journal of the American Chemical Society.</a></p>
  </li>
  <li>
    <p><a href="https://www.science.org/doi/10.1126/science.abo0058">Rohrbach, Simon, et al. “Digitization and validation of a chemical synthesis literature database in the ChemPU.” Science 377.6602 (2022): 172-180.</a></p>
  </li>
</ul>

<h3 id="automated-chemistry-workflows">
<a class="anchor" href="#automated-chemistry-workflows" aria-hidden="true"><span class="octicon octicon-link"></span></a>Automated chemistry workflows</h3>

<ul>
  <li>
    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.accounts.2c00220">Seifrid, Martin, et al. “Autonomous Chemical Experiments: Challenges and Perspectives on Establishing a Self-Driving Lab.” Accounts of Chemical Research (2022): e0229862-131.</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/10.1021/acscentsci.2c00207">Nambiar, Anirudh MK, et al. “Bayesian Optimization of Computer-Proposed Multistep Synthetic Routes on an Automated Robotic Flow Platform.” ACS Central Science (2022).</a></p>
  </li>
  <li>
    <p><a href="https://pubs.acs.org/doi/full/10.1021/acs.accounts.0c00674">Wilbraham, Liam, S. Hessam M. Mehr, and Leroy Cronin. “Digitizing chemistry using the chemical processing unit: from synthesis to discovery.” Accounts of Chemical Research 54.2 (2020): 253-262.</a></p>
  </li>
  <li>
    <p><a href="https://www.sciencedirect.com/science/article/pii/S135964461300069X">Godfrey, Alexander G., Thierry Masquelin, and Horst Hemmerle. “A remote-controlled adaptive medchem lab: an innovative approach to enable drug discovery in the 21st Century.” Drug Discovery Today 18.17-18 (2013): 795-802.</a></p>
  </li>
</ul>

<p>Account of Eli Lilly and Company’s ASL (Automated Synthesis Lab)</p>

<h3 id="dna-encoded-libraries">
<a class="anchor" href="#dna-encoded-libraries" aria-hidden="true"><span class="octicon octicon-link"></span></a>DNA-encoded Libraries</h3>

<ul>
  <li>
<a href="https://www.nature.com/articles/nchembio.211">Matthew Clark, et. al. DNA-encoded small-molecule libraries (DEL)</a>. <a href="https://cen.acs.org/articles/95/i25/DNA-encoded-libraries-revolutionizing-drug.html">C&amp;EN article on the topic</a>
</li>
</ul>

<p>New form of storing huge amounts of molecule related data using DNA. Made partially possible by low cost of DNA sequencing. Each molecule in the storage is attached with a DNA strand which encode information about its recipe.</p>

<ul>
  <li><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jmedchem.0c00452">Follow up to the work with Machine Learning for hit finding.</a></li>
</ul>

<p>DNA encodings for discovery of novel small-molecule protein inhibitors. Outline a process for building a ML model using DEL. Compare graph convolutions to random forest for classification tasks with application to protein target binding. Graph models seemed to achieve high hit rate comapred to random forest. Apply diversity, logistical, structural filtering to search for novel candidates. First work to use GCN for hit searching.</p>

<ul>
  <li>
    <p><a href="https://www.nature.com/articles/s42004-020-00374-1?error=cookies_not_supported&amp;code=2d1394f8-2e1b-46ef-b926-9441292aea56">Martín, A., et al. (2020). “Navigating the DNA encoded libraries chemical space.” Communications Chemistry 3(1).</a></p>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/60fac8718804431689e3155b">Zabolotna, Y., Pikalyova, R., Volochnyuk, D., Horvath, D., Marcou, G., &amp; Varnek, A. (2021). Exploration of the chemical space of DNA-encoded libraries.</a></p>
  </li>
</ul>

<h2 id="code--packages">
<a class="anchor" href="#code--packages" aria-hidden="true"><span class="octicon octicon-link"></span></a>Code / Packages:</h2>

<ul>
  <li>
    <p><a href="https://github.com/MolecularAI">Molecular AI department at AstraZeneca R&amp;D</a></p>
  </li>
  <li>
    <p><a href="https://github.com/rinikerlab/GHOST">GHOST: Generalized threshold shifting procedure</a>. <a href="https://pubs.acs.org/doi/10.1021/acs.jcim.1c00160">Paper</a>. <a href="https://greglandrum.github.io/rdkit-blog/exploratory/machinelearning/2021/12/23/ternary-ghost.html#Synthetic-datasets">Blogpost</a></p>
  </li>
</ul>

<p>Automates the selection of decision threshold for imbalanced classification task. The assumption for this method to work is the similar characteristics (like imbalance ratio) of training and test data.</p>

<ul>
  <li>
<a href="https://arxiv.org/abs/1811.12823">MOSES - Benchmarking platform for generative models (PyTorch Implementation)</a>. <a href="https://github.com/molecularsets/moses">Github</a>
</li>
</ul>

<p>Benchmarking platform to implement molecular generative models. It also provides a set of metrics to evaluate the quality and diversity of the generated molecules. A benchmark dataset (subset of ZINC) is provided for training the models.</p>

<ul>
  <li>
<a href="https://chemrxiv.org/articles/preprint/REINVENT_2_0_an_AI_Tool_for_De_Novo_Drug_Design/12058026/1">Reinvent 2.0 - an AI tool forr de novo drug design</a>. <a href="https://github.com/MolecularAI/Reinvent">Github</a>
</li>
</ul>

<p>Production-ready tool for de novo design from Astra Zeneca. It can be effectively applied on drug discovery projects that are striving to resolve either exploration or exploitation problems while navigating the chemical space. Language model with SMILE  output and trained by “randomizing” the SMILES representation of the input data. Implement reinforcement-leraning for directing the model towards relevant area of interest.</p>

<ul>
  <li>
    <p><a href="https://arxiv.org/abs/1806.03146">Schnet by Jacobsen et. al. (Neural message passing)</a>. <a href="https://github.com/atomistic-machine-learning/G-SchNet">Github</a>. <a href="https://schnetpack.readthedocs.io/en/stable/tutorials/tutorial_03_force_models.html">Tutorial</a></p>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/articles/OpenChem_A_Deep_Learning_Toolkit_for_Computational_Chemistry_and_Drug_Design/12691943/1">OpenChem</a>. <a href="https://github.com/Mariewelt/OpenChem">Github</a></p>
  </li>
  <li>
    <p><a href="https://github.com/deepchem/deepchem">DeepChem (Tensorflow)</a>. <a href="https://deepchem.io">Website</a></p>
  </li>
</ul>

<p>DeepChem aims to provide a high quality open-source toolchain that democratizes the use of deep-learning in drug discovery, materials science, quantum chemistry, and biology - from Github</p>

<ul>
  <li><a href="https://github.com/chemprop/chemprop">ChemProp (Pytorch)</a></li>
</ul>

<p>Github repository for implmenting message passing neural networks for molecular property prediction as described in the paper <a href="https://pubs.acs.org/doi/abs/10.1021/acs.jcim.9b00237">Analyzing Learned Molecular Representations for Property Prediction</a> by Yang et. al.</p>

<ul>
  <li><a href="https://github.com/chainer/chainer-chemistry">Chainer-Chemistry</a></li>
</ul>

<p>“Chainer Chemistry is a deep learning framework (based on Chainer) with applications in Biology and Chemistry. It supports various state-of-the-art models (especially GCNN - Graph Convolutional Neural Network) for chemical property prediction” - from their Github repo introduction</p>

<ul>
  <li>
    <p><a href="https://github.com/Bibyutatsu/FastJTNNpy3">FastJTNN - python 3 version of the JT-NN</a></p>
  </li>
  <li>
    <p><a href="https://arxiv.org/abs/2003.03123">DimeNet++  - extension of Directional message pasing working (DimeNet)</a>. <a href="https://github.com/klicperajo/dimenet">Github</a></p>
  </li>
  <li>
    <p><a href="https://github.com/mjwen/bondnet">BondNet - Graph neural network model for predicting bond dissociation energies, considers both homolytic and heterolytic bond breaking</a>. <a href="https://github.com/mjwen/bondnet">Github</a></p>
  </li>
  <li>
    <p><a href="https://arxiv.org/pdf/1902.08408.pdf">PhysNet</a></p>
  </li>
  <li>
    <p><a href="https://github.com/ETHmodlab/BIMODAL">RNN based encoder software</a></p>
  </li>
  <li>
    <p><a href="https://duartegroup.github.io/autodE/">AutodE</a></p>
  </li>
  <li>
    <p><a href="https://singroup.github.io/dscribe/latest/">DScribe</a></p>
  </li>
  <li>
    <p><a href="https://github.com/ReactionMechanismGenerator/RMG-Py">RMG - Reaction Mechanism Generator</a></p>
  </li>
</ul>

<p>Tool to generate chemical reaction networks. Includes Arkane, package for calculating thermodynamics from quantum mechanical calculations.</p>

<ul>
  <li><a href="https://pyepal.readthedocs.io/en/latest/?badge=latest">PyePAL</a></li>
</ul>

<p>Active learning approach to efficiently and confidently identify the Pareto front with any regression model that can output a mean and a standard deviation.</p>

<ul>
  <li><a href="https://github.com/rxn4chemistry/rxnfp">rxnfp</a></li>
</ul>

<p>Github repository to generate chemical reaction fingerprints from reaction SMILES.</p>

<ul>
  <li><a href="https://github.com/cbouy/mols2grid">mols2grid</a></li>
</ul>

<p>Interactive chemical viewer for small molecules (RDKit wrapper)</p>

<ul>
  <li><a href="https://github.com/wjm41/molplotly">molplotly</a></li>
</ul>

<p>Spotfire like capabilities to jupyter notebook.</p>

<h2 id="datasets--chemical-libraries">
<a class="anchor" href="#datasets--chemical-libraries" aria-hidden="true"><span class="octicon octicon-link"></span></a>Datasets &amp; Chemical libraries</h2>

<p><strong>Molecule datasets</strong></p>

<ul>
  <li>
    <p>PubChem: public sourced molecules</p>
  </li>
  <li>
    <p>ChEMBL: bioactive molecules (most synthetic)</p>
  </li>
  <li>
    <p>ZINC: collection of synthetic molecules (not all are bioactive)</p>
  </li>
  <li>
    <p>QM 7/8/9: small molecules having not more than 7/8/9 heavy atoms</p>
  </li>
  <li>
    <p><a href="https://chemrxiv.org/engage/chemrxiv/article-details/617aa2467a002162403d71f0">Papyrus</a></p>
  </li>
  <li>
    <p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00478-9">COCONUT</a>: NP 400k there are some which are not NP</p>
  </li>
  <li>
    <p><a href="https://mcule.com/database/">Mcule</a>: Used in DEL enumerations</p>
  </li>
  <li>
    <p><a href="https://go.drugbank.com/">DrugBank</a></p>
  </li>
  <li>
    <p><a href="https://www.nature.com/articles/s41597-022-01390-7">QMugs</a></p>
  </li>
</ul>

<p>QMugs (Quantum mechanical properties of drug-like molecules) collection comprises quantum mechanical properties of more than 665 k biologically and pharmacologically relevant molecules extracted from the ChEMBL database, totaling 2M conformers.</p>

<p><strong>Reaction Datasets</strong></p>

<ul>
  <li>USPTO</li>
  <li>Pistachio</li>
  <li>Reaxys</li>
  <li>Open Reaction Database</li>
</ul>

<p><strong>Commericial (building block) vendors</strong></p>

<ul>
  <li>
    <p>eMolecules building blocks</p>
  </li>
  <li>
    <p>Enamine REAL Space</p>
  </li>
  <li>
    <p>WuXi GalaXi space</p>
  </li>
  <li>
    <p>Otava’s CHEMriya</p>
  </li>
</ul>

<h2 id="helpful-utilities">
<a class="anchor" href="#helpful-utilities" aria-hidden="true"><span class="octicon octicon-link"></span></a>Helpful utilities:</h2>

<ul>
  <li>
<a href="https://github.com/rdkit/rdkit">RD-Kit</a>
    <ul>
      <li><a href="https://colab.research.google.com/drive/16T6ko0YE5WqIRzL4pwW_nufTDn7F3adw">Get Atom Indices in the SMILE:</a></li>
      <li><a href="https://github.com/datamol-org/datamol">Datamol for manipulating RDKit molecules</a></li>
    </ul>
  </li>
  <li>
    <p><a href="https://paperswithcode.com/sota/formation-energy-on-qm9">Papers with code benchmark for QM9 energy predictions</a></p>
  </li>
  <li>
    <p><a href="https://github.com/molecularsets/moses">MOSES: Molecular generation models benchmark</a></p>
  </li>
  <li>
<a href="https://tdcommons.ai">Therapeutics Data Commons</a>
“Therapeutics Data Commons is an open-science platform with AI/ML-ready datasets and learning tasks for therapeutics, spanning the discovery and development of safe and effective medicines. TDC also provides an ecosystem of tools, libraries, leaderboards, and community resources, including data functions, strategies for systematic model evaluation, meaning”</li>
</ul>

  </div><a class="u-url" href="/blog_fastpages/chemical-science/machine-learning/resources/2022/12/28/Computational_drug_discovery.html" hidden></a>
</article>
      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/blog_fastpages/"></data>

  <div class="wrapper">

    <div class="footer-col-wrapper">
      <div class="footer-col">
        <p class="feed-subscribe">
          <a href="http://pgg1610.github.io/blog_fastpages/feed.xml">
            <svg class="svg-icon orange">
              <use xlink:href="/blog_fastpages/assets/minima-social-icons.svg#rss"></use>
            </svg><span>Subscribe</span>
          </a>
        </p>
      </div>
      <div class="footer-col">
        <p>Blog on data &amp; science</p>
      </div>
    </div>

    <div class="social-links"><ul class="social-media-list"><li>
  <a rel="me" href="https://github.com/pgg1610" target="_blank" title="github">
    <svg class="svg-icon grey">
      <use xlink:href="/blog_fastpages/assets/minima-social-icons.svg#github"></use>
    </svg>
  </a>
</li>
<li>
  <a rel="me" href="https://twitter.com/mepgg" target="_blank" title="twitter">
    <svg class="svg-icon grey">
      <use xlink:href="/blog_fastpages/assets/minima-social-icons.svg#twitter"></use>
    </svg>
  </a>
</li>
<li>
  <a rel="me" href="https://linkedin.com/pushkarghanekar" target="_blank" title="linkedin">
    <svg class="svg-icon grey">
      <use xlink:href="/blog_fastpages/assets/minima-social-icons.svg#linkedin"></use>
    </svg>
  </a>
</li>
<li>
  <a rel="me" href="https://instagram.com/pgg1610" target="_blank" title="instagram">
    <svg class="svg-icon grey">
      <use xlink:href="/blog_fastpages/assets/minima-social-icons.svg#instagram"></use>
    </svg>
  </a>
</li>
</ul>
</div>

  </div>

</footer>
</body>

</html>
